sdks/python/build.gradle

```diff
@@ -174,6 +174,7 @@ def portableWordCountTask(name, streaming) {
               "--experiments=worker_threads=100",
               "--parallelism=2",
               "--shutdown_sources_on_final_watermark",
+              "--environment_type=LOOPBACK",
```

Comment:
Lets add environment_type and environment_config as option to gradle task and keep the default behavior of not passing those value.
This is to run the example which resemble production more closely with flexibility of using different environments.

Reply:
Alright, I changed it